packages/slice-machine/src/modules/environment/index.ts

```diff
@@ -49,8 +49,19 @@ export const getUpdateVersionInfo = (
   return store.environment.env.updateVersionInfo;
 };
 
-export const getStorybookUrl = (state: SliceMachineStoreType) => {
-  return state.environment.env.manifest.storybook || null;
+export const getCurrentVersion = (store: SliceMachineStoreType): string => {
+  const { currentVersion } = getUpdateVersionInfo(store);
+  return currentVersion;
+};
+
+export const getStorybookUrl = (
+  store: SliceMachineStoreType
+): string | null => {
+  return store.environment.env.manifest.storybook || null;
+};
+
+export const getUserID = (store: SliceMachineStoreType): string | null => {
+  return store.environment.env.userId || null;
```

Comment:
I think you can return 'store.environment.env.userId' directly as this is already null | string :)

Reply:
Ignore this comment because you will remove this selector :)

Reply:
done